inst/rmarkdown/templates/gigascience_article/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,548 @@
+#   - Data Note
+#     - http://academic.oup.com/gigascience/pages/data_note
+#   - Research
+#     - http://academic.oup.com/gigascience/pages/research
+#   - Review
+#     - http://academic.oup.com/gigascience/pages/review
+#   - Technical Note
+#     - http://academic.oup.com/gigascience/pages/technical_note
+papercat: Technical Note
+
+# abstract: abstract structured according to instructions for paper category
+abstract: |
+  The Abstract (250 words maximum) should be structured according to the instructions for the corresponding 'papercat' (paper category, see https://academic.oup.com/gigascience/pages/instructions_to_authors).
+
+# keywords: keywords presented as a semicolon-separated list below the abstract  
+keywords:
+  - Keyword 1
+  - keyword 2
+  - keyword 3 (Three to ten keywords representing the main content of the article)
+
+# keypoints: key points presented as a bullet list in a box at the top of the second page
+keypoints:
+  - This is the first point
+  - This is the second point
+  - One last point.
+
+# highlighting-macros: list of {latex_command+}
+#   - This may be for syntax-highlighting source-code, but I'm not sure
+#   - Value is a list; each list-member is a TeX snippet
+#   - This is included just before 'header-includes'
+
+# header-includes: list of {latex_command+}
+#   - Value is a list; each list-member is a TeX snippet
+#   - You can use 'header-includes' to set or override some TeX settings, e.g.,
+#   - This is included just before 'preamble'
+#   - concerning specific keys:
+#     - journal: Only "gigascience" and "general" are implemented now.
+#     - usepackage: use packages that are not declared in the TeX template used by pandoc
+#     - bibliographystyle: apples to the natbib TeX package
+#       - see http://ctan.math.washington.edu/tex-archive/macros/latex/contrib/natbib/natbib.pdf
+header-includes:
+  - \journal{gigascience}
+  - \usepackage{natbib}
+#   # the next line is part of the document-class, oup-contemporary
+#   - \bibliographystyle{vancouver-authoryear}
+
+# preamble: Tex literal (newlines will be stripped out)  
+#   - Value is a TeX snippet that will be collapsed to a single line
+# preamble: |
+#  \setcounter{errorcontextlines}{4}
+#  \setcounter{secnumdepth}{2}
+
+# document-class - the TeX document class - for GigaScience, muct be oup-contemporary
+document-class: oup-contemporary
+
+# paper: 'a4paper' or 'letterpaper'
+paper: letterpaper
+#paper: a4paper
+
+# reftype: 'num-refs' or 'alpha-refs', but 'alpha-refs' may not work ...
+reftype: num-refs
+#reftype: alpha-refs
+
+# pandoc_args: arguments to pandoc
+#   - enable with '+' and disable with '-'
+#       - see: https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html#pandoc_markdown
+#   - concerning specific keys:
+#       raw_tex:
+#         - allow direct inclusion of TeX
+#         - see https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html#raw-tex`
+pandoc_args: +raw_tex
```

Comment:
I have never heard of this top-level option. Could you point me to its documentation?

Usually `pandoc_args` is a sub-option for an R Markdown output format: https://bookdown.org/yihui/rmarkdown/output-formats.html

Reply:
[http://pandoc.org/MANUAL.html#raw-htmltex](http://pandoc.org/MANUAL.html#raw-htmltex)
The link in the comment worked at the time that I originally submitted the PR
[https://web.archive.org/web/20171220122438/https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html#raw_tex](https://web.archive.org/web/20171220122438/https://rmarkdown.rstudio.com/authoring_pandoc_markdown.html#raw_tex)

Reply:
Yes, I was aware of the `raw_tex` extension. I was asking about the usage of `pandoc_args` as a _top-level_ field in YAML. I have never seen any documentation on that. That is one problem. The other problem is that the Markdown extension is typically specified through the `md_extensions` argument of an output format instead of `pandoc_args`: https://bookdown.org/yihui/rmarkdown/html-document.html#markdown-extensions

Reply:
This
```
#pandoc_args: +raw_tex
output: 
  rticles::gigascience_article:
    +raw_tex
```
results in this:
```
Error in eval(parse(text = name)) : object 'raw_tex' not found
Calls: <Anonymous> ... create_output_format -> create_output_format_function -> eval -> eval
Execution halted
```
whereas this
```
pandoc_args: +raw_tex
output: 
  rticles::gigascience_article:
#    +raw_tex
```
results in this:
```
/usr/bin/pandoc +RTS -K512m -RTS Instructions.utf8.md --to latex --from markdown+ascii_identifiers+tex_math_single_backslash-autolink_bare_uris --output Instructions.tex --template /zpdrone/drone/home/art/R/x86_64-pc-linux-gnu-library/3.4/rticles/rmarkdown/templates/gigascience_article/resources/template.tex --highlight-style tango --pdf-engine pdflatex
```
So, **how would you like me to specify `raw_tex` differently?**

Thank you.

Reply:
I pointed out the documentation in my previous reply. I said the argument name was `md_extensions`, e.g.,

```yaml
output: 
  rticles::gigascience_article:
    md_extensions: +raw_tex
```

And you didn't answer my question on the use of `pandoc_args: +raw_tex`. I was asking because I was hoping to correct the misinformation if you tell me where you learned it.

Reply:
Okay, thank you for the explicit example. I am sorry that I did not put the pieces of information together.

IT WORKS!  Thank you.

The reference is all that I have to remember by.  I think that I kept trying things and it happened to work.  I **vastly** prefer your approach here!